From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Mon, 22 Feb 2021 10:56:48 -0800
Subject: ANDROID: gki_defconfig: enable CONFIG_MMC_CRYPTO

This is needed to support eMMC inline encryption hardware.

Bug: 161256007
Change-Id: Ib8a40363a77f787111632348ec3662c6f7550241
Signed-off-by: Eric Biggers <ebiggers@google.com>
(cherry picked from commit 018ad7e673b83a75936fc59c1be7fecc43aace21)
Signed-off-by: Palmer Dabbelt <palmerdabbelt@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -458,6 +458,7 @@ CONFIG_TYPEC_UCSI=y
 CONFIG_MMC=y
 # CONFIG_PWRSEQ_EMMC is not set
 # CONFIG_PWRSEQ_SIMPLE is not set
+CONFIG_MMC_CRYPTO=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_LEDS_CLASS_FLASH=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -413,6 +413,7 @@ CONFIG_TYPEC_UCSI=y
 CONFIG_MMC=y
 # CONFIG_PWRSEQ_EMMC is not set
 # CONFIG_PWRSEQ_SIMPLE is not set
+CONFIG_MMC_CRYPTO=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_LEDS_CLASS_FLASH=y
